src/app/qgsstatusbarcoordinateswidget.cpp

```diff
@@ -194,6 +208,52 @@ void QgsStatusBarCoordinatesWidget::dizzy()
   mMapCanvas->setTransform( matrix );
 }
 
+void QgsStatusBarCoordinatesWidget::contributors()
+{
+  if ( !mMapCanvas )
+  {
+    return;
+  }
+  QString myFileName = QgsApplication::pkgDataPath() + QStringLiteral( "/resources/data/contributors.json" );
+  QFileInfo myFileInfo = QFileInfo( myFileName );
+  QgsVectorLayer *layer = new QgsVectorLayer( myFileInfo.absoluteFilePath(),
+      "QGIS Contributors", QStringLiteral( "ogr" ) );
+  // Register this layer with the layers registry
+  QgsProject::instance()->addMapLayer( layer );
+  layer->setAutoRefreshInterval( 500 );
+  layer->setAutoRefreshEnabled( true );
```

Comment:
What's this for? The layer data shouldn't be changing, unless I'm missing something?

Reply:
The data no, but the style yes, cf https://twitter.com/timlinux/status/1033784351740198913

Reply:
@nyalldawson yes the little wheel icons rotate pleasingly on the screen because of that....